tests/end_to_end/definitions/kcf_custom_image_definitions.py

```diff
@@ -50,13 +50,6 @@ def save_to_file(filename, data):
     root_path="testing/ff",
 )
 
-redis = ff.register_redis(
```

Comment:
why remove?

Reply:
it wasn't being used. the offline and online use azure blob store.